department-of-veterans-affairs/caseflow
# TYLERB/Membership-request-mailer-uat-testing-fix

Resolves #{github issue number}

### Description
Adjusted the recipient email for uat testing to always be set to the BID_Appeals_UAT@bah.com email address for the membership request mailer methods.

### Acceptance Criteria
- [ ] The membership request emails should always be sent to BID_Appeals_UAT@bah.com for any membership request mailer email 

### Testing Plan
1. Create a request and verify that the email is sent to the BID_Appeals_UAT@bah.com for all types of emails
2. Login as an admin user
3. Approve or deny the request and verify that the confirmation email is sent to BID_Appeals_UAT@bah.com for both approval and denial.
